webapp/src/container.tsx

```diff
@@ -98,7 +107,7 @@ export class SideDocs extends data.Component<ISettingsProps, {}> {
         if (!docsUrl) return null;
 
         return <div>
-            <button id="sidedocstoggle" role="button" aria-label={state.sideDocsCollapsed ? lf("Expand the side documentation") : lf("Collapse the side documentation")} className="ui icon button" onClick={() => this.toggleVisibility() }>
+            <button id="sidedocstoggle" role="button" aria-label={state.sideDocsCollapsed ? lf("Expand the side documentation") : lf("Collapse the side documentation. Press tab to navigate into the documentation.")} className="ui icon button" onClick={() => this.toggleVisibility() }>
```

Comment:
Not sure this is necessary, I think it should be obvious that tabbing is going to navigate.